### PR TITLE
fix(config): network case

### PR DESCRIPTION
**Pull request by gurinderu**
## Description
Change the case to the lower case for network enum.

## Checklist
- [ ] The code follows the project's coding conventions and style guidelines.
- [ ] All tests related to the changes have passed successfully.
- [ ] Documentation has been updated to reflect the changes (if applicable).
- [ ] All new and existing unit tests have passed.
- [ ] I have self-reviewed my code and ensured its quality.
- [ ] I have added/updated necessary comments to aid understanding.

## Reviewer Checklist
- [ ] Code has been reviewed for quality and adherence to [guidelines](https://doc.rust-lang.org/1.0.0/style/README.html).
- [ ] Tests have been reviewed and are sufficient to validate the changes.
- [ ] Documentation has been reviewed and is up to date.
- [ ] Any questions or concerns have been addressed.

### CODE REVIEW

**Review by mikevoronov — APPROVED** [review]
LGTM, but I suggest waiting also for @folex @justprosh approve.

**Comment by mikevoronov** [comment]
+ @shamsartem

File: crates/server-config/src/node_config.rs [hunk 1]
```diff
@@ -182,6 +182,7 @@ pub struct UnresolvedNodeConfig {
 
 #[serde_as]
 #[derive(Clone, Deserialize, Serialize, Debug, Default, PartialEq)]
+#[serde(rename_all = "lowercase")]
```
Comment (folex) [inline crates/server-config/src/node_config.rs:185]: Is it case insensitive on deserialization? it's kinda weird to care about that. I think `rename_all = lowercase` might do that, can you check?

Comment (gurinderu) [reply crates/server-config/src/node_config.rs:185]: it is a case sensitive

Comment (folex) [reply crates/server-config/src/node_config.rs:185]: well, if Rust / serde has no built in way of making it case insensitive, then I guess let's leave it lowercase

**Pull request merged**